src/components/Link/Link.tsx

```diff
@@ -27,16 +28,17 @@ export interface BaseProps {
 export interface Props extends BaseProps {}
 type CombinedProps = Props & WithAppProviderProps;
 
-function Link({
-  url,
-  children,
-  onClick,
-  external,
-  id,
-  monochrome,
-  polaris,
-}: CombinedProps) {
-  const className = classNames(styles.Link, monochrome && styles.monochrome);
+function Link(
```

Comment:
```suggestion
export function Link(
```

I think this should fix it

Reply:
Is this issue occurring on master, too?

https://github.com/Shopify/polaris-react/blob/master/src/components/Link/Link.tsx#L30

Reply:
It's not, I think it's a ts issue related to "statics"